### ISSUE TITLE
Playlists can not be invoked in Kodi 18

### ISSUE THREAD

**Issue by bill-orange** [m0ngr31/kodi-alexa#177]
Over the last couple of days I have observed that playlists don't work right in Kodi 18 (alpha). 

 Jingai's thought was that a parameter in the playlist function was depreciated in Kodi 18. 

 On Kodi 17 playlist requests work perfectly. On Kodi 18 they are broken. 

I have posted this issue on the LibreElec forum.  Hopefully, they will give us some direction as to what changed.

 http://forum.kodi.tv/showthread.php?tid=298461&page=125

**Comment by bill-orange** [comment]
Looks like others are also noting a problem.

http://forum.kodi.tv/showthread.php?tid=315249

The guy that maintains Yatse has some sort of a work-around.

**Comment by jingai** [comment]
That isn't a work-around though -- that's exactly what we already do if you specify `shuffle playlist foo`.

Basically, this is a bug in Kodi, it seems.

**Comment by jingai** [comment]
I'll keep this Issue open here just to remind us to keep an eye on it, but @bill-orange, you should do as suggested and post the issue on Kodi's trac.

**Comment by bill-orange** [comment]
Indeed. No reply on the main Kodi forum yet. 

http://forum.kodi.tv/showthread.php?tid=298461&page=125

**Comment by bill-orange** [comment]
This thread may pertain to the problem.

https://forum.kodi.tv/showthread.php?tid=318494

**Comment by jingai** [comment]
Unfortunately not -- we already use POST.

**Comment by mcl22** [comment]
Any news on that one? I recently switched to kodi 18 and well can't have Alexa play playlists anymore :)

**Comment by jingai** [comment]
Nope, no news.  It's not clear if it's an intentional change or not either.  I suspect it's not, but we haven't seen a dev say one way or the other.

**Comment by bill-orange** [comment]
I rattled the cage on my earlier post to the JSON-RPC forum.

https://forum.kodi.tv/showthread.php?tid=315249

I reverted to Kodi 17.

**Comment by bill-orange** [comment]
Had there been any motion on this issue.  I am not aware of any.

**Comment by jingai** [comment]
Not that I know of :(

**Comment by zarianu** [comment]
Hello dear sirs, any updates on this issue?

It seems like there's some workaround, no? 
https://forum.kodi.tv/showthread.php?tid=315249&pid=2694920#pid2694920

P.S. Thanks for your huge job creating this piece of software!
